contrib/bash/completion.bash

```diff
@@ -16,94 +16,76 @@ _btrbk_init_cmds()
   for ((i = 1; i < cword; i++)); do
     [[ ${words[i]} != -* ]] && cmds+=(${words[i]})
   done
+
+  return 0
 }
 
 _btrbk()
 {
-  local cur prev words cword split
+  local cur prev words cword split cmds
   _init_completion -s || return
+  _btrbk_init_cmds || return
 
   case "$prev" in
     '-c' | '--config')
       _filedir
-      return
       ;;
     '--exclude')
-      return
       ;;
     '-l' | '--loglevel')
       COMPREPLY=($(compgen -W 'error warn info debug trace' -- "$cur"))
-      return
       ;;
     '--format')
       COMPREPLY=($(compgen -W 'table long raw' -- "$cur"))
-      return
       ;;
     '--lockfile')
       _filedir
-      return
       ;;
     '--override')
-      return
       ;;
   esac
-
   $split && return
 
-  local cmds
-  _btrbk_init_cmds
+  if [[ $cur == -* ]]; then
+    COMPREPLY=($(compgen -W '$(_parse_help "$1")' -- "$cur"))
+    [[ $COMPREPLY == *= ]] && compopt -o nospace
+  else
+    if [[ ! -v 'cmds[0]' ]]; then
+      COMPREPLY=($(compgen -W 'run dryrun snapshot resume prune archive clean stats list usage origin diff ls' -- "$cur"))
+    fi
+  fi
 
   case "${cmds[0]}" in
     'archive')
       # <source>
       if [[ ! -v 'cmds[1]' ]]; then
+        echo 'a' 1>&2
```

Comment:
is this some required completion magic, or just forgotten debug code?

Reply:
It's just forgotten debug code, thanks, I'll remove it.